src/classes.scss

```diff
@@ -19,7 +20,8 @@
 @import './header/classes';
 @import './icon/classes';
 @import './information-panel/classes';
-@import './inputs/classes';
+@import './label/classes'; // Needs to be above Input
```

Comment:
If this line is below `/input/` we get this:
![Screen Shot 2022-04-29 at 4 38 30 PM](https://user-images.githubusercontent.com/849817/166065973-d3f5b21b-de3b-4fa2-9551-b9af5b79ac17.png)

Not a fan that this is no longer alphabetical.  Suggestions?

Reply:
If order starts to matter, it is wrong. We might have issues of specificty and import order..

Not sure I understand that `label` part at all.. It should be named differently, I think, cause there is a bunch of different mixins starting from container, ending up with certain placement..

Maybe it would make sense to separate those even further?

Reply:
If order starts to matter, it is wrong. We might have issues of specificty and import order..

Not sure I understand that `label` part at all.. It should be named differently, I think, cause there is a bunch of different mixins starting from container, ending up with certain placement..

Maybe it would make sense to separate those even further?

Reply:
I could rename `/label` to `/input-container/` if that's what you're suggesting.

Reply:
No, we need to investigate what causes issues with having different import order. Because in React we have no idea in what order CSS files will be imported, nor we can control that.

Reply:
`input-with-label` imports `'../label/index';`
classes.scss in Label folder imports ``'../input/index'

Maybe this loop is causing failures?

Reply:
Loop is removed & ordering has been fixed by doing [this](https://github.com/iTwin/iTwinUI/pull/620/files#r865318081).

Reply:
bad fix, but...
Can we still rename label to something else? I barely find connection between name and content in there...

Reply:
How about `/input-container/`?

Reply:
I guess its better, but still.. there are 2 mixins about input container.. there are many more for labels, icons, statuses, etc..
Seems like a file with a bunch of stuff which is generic to all the inputs?

Reply:
Renamed to `/input-container/`.

Reply:
SASS file name should be the same as folder name.